.github/workflows/linux-raspbian.yml

```diff
@@ -0,0 +1,140 @@
+name: Raspbian
+
+on: [push, pull_request]
+
+env:
+  APPIMAGEVER: 13
+  PPAQTVER: 515
+  QTIFWVER: 4.5.1
+  QTVER: 5.15.2
+  NJOBS: 4
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+      fail-fast: false
+      matrix:
+       include:
+         - dockerimg: ubuntu:focal
+           compiler: gcc
+           upload: 1
```

Comment:
```upload: 1``` should be ```upload: 0``` for now, so it does not collide with x86 desktop builds.

Reply:
I had debated on making a mac_arm.yml action. Do you think this would be better?

Reply:
For me, differencing raspbian from other distros is ok, but for Mac is the same OS but with different architectures, IMO is ok to be in the same file, as for example in the case of the [MSYS build](https://github.com/webcamoid/webcamoid/blob/master/.github/workflows/windows-msys.yml).